build.sbt

```diff
@@ -188,7 +188,7 @@ lazy val library =
     }
     val akkaHttp            = "com.typesafe.akka"                     %% "akka-http"             % Version.akkaHttp
     val akkaHttpJacksonJava = "com.typesafe.akka"                     %% "akka-http-jackson"     % Version.akkaHttp
-    val akkaStream          = "com.typesafe.akka"                     %% "akka-stream"           % Version.akka
+    val akkaStream          = "com.typesafe.akka"                     %% "akka-stream"           % Version.akka % Provided
```

Comment:
Not here, at the definition site, please, but where `akkaStream` is used.

Reply:
Thanks! I've fixed it.